obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: 'Build SRPM'
+  command: "tito build --srpm --offline"
+  args:
+    chdir: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}"
+  register: srpm_build
+
+- debug:
```

Comment:
this is still not reached when `command` fails, as it (the `command` module) will directly bail out. but I think we can actually drop that `debug` alltogether, as `command` will output the error cleanly if there was one.